content/toolsAndResources/assetLibraries/postProcessLibrary/edgeDetectionPP.md

```diff
@@ -0,0 +1,70 @@
+---
+title: Edge Detection
+image: 
+description: Check out the edge detection post process in your Babylon.js scenes.
+keywords: babylon.js, tools, resources, assets, post process, edge detection, outline, toon
+further-reading:
+video-overview:
+video-content:
+---
+
+
+## Using the Edge detection post-process
+
+With this Edge Detection post-process, you can create full-scene outlines in a style suited for comics or games, which cannot be achieved with renderOutline, edgeRendering, or HighlightLayer.
+
+If you’re aiming for that look, this tutorial is for you.
+
+![Screenshot](/img/resources/edgeDetectionPP/EDPP.png)
+
+## How to use ?
+
+To use it, attach the post-process to your main camera to enable the effect.
+
+```
+// Creates the post process
+var edgeDetectionPostProcess = new BABYLON.EdgeDetectionPostProcess("EdgeDetection", camera);
+```
+<Playground id="#HBSUDW" title="Edge Detection Post Process Demo" description="Edge Detection Post Process Demo"/>
+
+## Going further
+
+You can adjust the thickness of the outlines like so:
+
+<!-- default 값 변경 필요 -->

```

Comment:
please remove any comments that are not in english

Reply:
Oops I'm sorry for that